spec/views/events.html.erb_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe "events/index", type: :view do
+  context "events page" do
+    it "lists events ordered by start asc in localtime" do
+      assign(:events, [
+               event1 = FactoryBot.create(:event),
+               event2 = FactoryBot.create(:event, start: "2019-09-08 03:00:00", finish: "2019-09-08 20:00:00", error: nil, success: true, created_at: "2019-09-08 20:00:00", updated_at: "2019-09-08 20:00:00"),
+               event3 = FactoryBot.create(:event, start: "2019-11-07 03:00:00", finish: "2019-11-07 20:00:00", error: nil, success: true, created_at: "2019-11-07 20:00:00", updated_at: "2019-11-07 20:00:00")
+             ])
+      render
+      expect(rendered).to have_css('tr.table-striped:nth-child(1) > td:nth-child(1)', text: event1.start.localtime.to_s(:db_twelve_hour))
+      expect(rendered).to have_css('tr.table-striped:nth-child(2) > td:nth-child(1)', text: event2.start.localtime.to_s(:db_twelve_hour))
+      expect(rendered).to have_css('tr.table-striped:nth-child(3) > td:nth-child(1)', text: event3.start.localtime.to_s(:db_twelve_hour))
```

Comment:
I think we need to insert into the database in an order other than the final ascending order. Did this test fail before the code change?

Reply:
I added this test to test the view and more specifically the localtime it does not check the order because I assign the events in the specific order I expect them to display.

Reply:
We need a test that will also fail if we don't make the code change. Currently it succeeds with the `Event.all` implementation.

Reply:
I understand that it succeeds. But in the first place we should enforce the order. The odd behavior we see in the database it might be from postgres as it is when you do a select. You must add an order if you want the records to return they way you expect them to. From the other side it might be an error from postgres the way we migrated the databases when we used the pg_loader gem. I dont know if it worth's to test something that we don't know where it starts from but we do know what we want the result to be in an ascending order.

Reply:
I will try to fake some records with unordered ids.

Reply:
@hackmastera thanks for insisting :) . Changing the creation order of the events will fail the test.